client/src/js/administration/components/__tests__/SampleRights.test.js

```diff
@@ -19,17 +19,31 @@ describe("<SampleRights />", () => {
         expect(wrapper).toMatchSnapshot();
     });
 
-    it("should call props.onChangeSampleGroup() when input changes", () => {
+    it("should call props.onChangeSampleGroup() when SelectBox is clicked", () => {
         const wrapper = shallow(<SampleRights {...props} />);
-        const e = {
-            target: {
-                value: "users_primary_group"
-            }
-        };
+
         wrapper
-            .find("InputError")
+            .find("SelectBox")
             .at(0)
-            .simulate("change", e);
+            .simulate("click");
+        expect(props.onChangeSampleGroup).toHaveBeenCalledWith("none");
+    });
+    it("should call props.onChangeSampleGroup() when SelectBox is clicked", () => {
+        const wrapper = shallow(<SampleRights {...props} />);
+
+        wrapper
+            .find("SelectBox")
+            .at(1)
+            .simulate("click");
+        expect(props.onChangeSampleGroup).toHaveBeenCalledWith("force_choice");
+    });
+    it("should call props.onChangeSampleGroup() when SelectBox is clicked", () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Test title is used multiple times in the same describe block.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5028429)